### PR TITLE
Added links to mdn docs

### PR DESCRIPTION
**Pull request by yepninja**
#32

### CODE REVIEW

File: tasks/start-template.html [hunk 1]
```diff
@@ -43,6 +43,9 @@ <h2 class="cssdb-feature-heading">
 		<div class="cssdb-feature-content">
 			<p class="cssdb-feature-specification">
 				<a class="cssdb-feature-specification-link" href="${feature.specification}">Specification</a>
+				${feature.mdn
+					? `<a class="cssdb-feature-mdn-link" href="${feature.mdn}">MDN Docs</a>`
```
Comment (nucliweb) [inline tasks/start-template.html]: If you create a new css class, you need add it into [`style.css`](https://github.com/csstools/cssdb/blob/gh-pages/style.css) from `gh-pages` branch.

Comment (jonathantneal) [reply tasks/start-template.html]: Yes, if we think it needs styling.

Comment (jonathantneal) [reply tasks/start-template.html]: And I agree with @nucliweb that we can add target blank to these.

```suggestion
					? `<a class="cssdb-feature-mdn-link" href="${feature.mdn}" target="_blank">MDN Docs</a>`
```

**Comment by jonathantneal** [comment]
This resolves #32

**Pull request merged**